src/Adapter/Presenter/Module/ModulePresenter.php

```diff
@@ -66,6 +66,11 @@ public function present($module)
         $attributes['price'] = $this->getModulePrice($attributes['price']);
         $attributes['starsRate'] = str_replace('.', '', round($attributes['avgRate'] * 2) / 2); // Round to the nearest 0.5
 
+        $moduleInstance = $module->getInstance();
+        if ($moduleInstance !== null && $moduleInstance->getMultistoreCompatibility() !== null) {
```

Comment:
You can get rid of this check
```suggestion
        if ($moduleInstance !== null) {
```